### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by Ifeanyi668** [Ilyasayunus/belajar-kolaborasi#6]
Thanks for your work but how can I install 
What are the pip install requirements 
Please assist

**Comment by beansbeanessssss** [comment]
I agree and where is video instructions I have installed java and python so far
